docs/references/users/create-argon2-user.md

```diff
@@ -0,0 +1 @@
+Create a new user. Password entered must be hashed with the the [Argon2](https://en.wikipedia.org/wiki/Argon2) algorithm. Use the [POST /users](/docs/server/users#usersCreate) endpoint to import plain text password.
```

Comment:
I would say `password provided` and not `entered`; we cannot assume that this is only used for client auth.

I also personally dislike the use of `import` here, because the endpoint is called create.

(Also you have `the the` here, reminder to run it through Grammarly XD)

Suggested format for all docs:
> Create a new user. Password provided must be hashed with the [Argon2](https://en.wikipedia.org/wiki/Argon2) algorithm. Use the [POST /users](/docs/server/users#usersCreate) endpoint to create users with a plain text password.